lib/board.rb

```diff
@@ -1,26 +1,33 @@
 class Board
-  attr_reader :board
+  attr_reader :visual
 
   def initialize
-    @board = {
-      A: %w[. . . . . .],
-      B: %w[. . . . . .],
-      C: %w[. . . . . .],
-      D: %w[. . . . . .],
-      E: %w[. . . . . .],
-      F: %w[. . . . . .],
-      G: %w[. . . . . .]
+    @visual = {
+      "A" => %w[. . . . . .],
+      "B" => %w[. . . . . .],
+      "C" => %w[. . . . . .],
+      "D" => %w[. . . . . .],
+      "E" => %w[. . . . . .],
+      "F" => %w[. . . . . .],
+      "G" => %w[. . . . . .]
     }
   end
 
-  def print_board
+  def print_visual
     "
     A | B | C | D | E | F | G \n
-    #{board[:A][5]} | #{board[:B][5]} | #{board[:C][5]} | #{board[:D][5]} | #{board[:E][5]} | #{board[:F][5]} | #{board[:G][5]}
-    #{board[:A][4]} | #{board[:B][4]} | #{board[:C][4]} | #{board[:D][4]} | #{board[:E][4]} | #{board[:F][4]} | #{board[:G][4]}
-    #{board[:A][3]} | #{board[:B][3]} | #{board[:C][3]} | #{board[:D][3]} | #{board[:E][3]} | #{board[:F][3]} | #{board[:G][3]}
-    #{board[:A][2]} | #{board[:B][2]} | #{board[:C][2]} | #{board[:D][2]} | #{board[:E][2]} | #{board[:F][2]} | #{board[:G][2]}
-    #{board[:A][1]} | #{board[:B][1]} | #{board[:C][1]} | #{board[:D][1]} | #{board[:E][1]} | #{board[:F][1]} | #{board[:G][1]}
-    #{board[:A][0]} | #{board[:B][0]} | #{board[:C][0]} | #{board[:D][0]} | #{board[:E][0]} | #{board[:F][0]} | #{board[:G][0]}"
+    #{visual["A"][5]} | #{visual["B"][5]} | #{visual["C"][5]} | #{visual["D"][5]} | #{visual["E"][5]} | #{visual["F"][5]} | #{visual["G"][5]}
+    #{visual["A"][4]} | #{visual["B"][4]} | #{visual["C"][4]} | #{visual["D"][4]} | #{visual["E"][4]} | #{visual["F"][4]} | #{visual["G"][4]}
+    #{visual["A"][3]} | #{visual["B"][3]} | #{visual["C"][3]} | #{visual["D"][3]} | #{visual["E"][3]} | #{visual["F"][3]} | #{visual["G"][3]}
+    #{visual["A"][2]} | #{visual["B"][2]} | #{visual["C"][2]} | #{visual["D"][2]} | #{visual["E"][2]} | #{visual["F"][2]} | #{visual["G"][2]}
+    #{visual["A"][1]} | #{visual["B"][1]} | #{visual["C"][1]} | #{visual["D"][1]} | #{visual["E"][1]} | #{visual["F"][1]} | #{visual["G"][1]}
+    #{visual["A"][0]} | #{visual["B"][0]} | #{visual["C"][0]} | #{visual["D"][0]} | #{visual["E"][0]} | #{visual["F"][0]} | #{visual["G"][0]}"
+  end
+
+  def update_board(letter, player)
```

Comment:
Awesome method for selecting the next available "." in the row the player selects, and updating the row with the player's piece.